Lib/functools.py

```diff
@@ -322,10 +403,14 @@ def __setstate__(self, state):
             raise TypeError(f"expected 4 items in state, got {len(state)}")
         func, args, kwds, namespace = state
         if (not callable(func) or not isinstance(args, tuple) or
-           (kwds is not None and not isinstance(kwds, dict)) or
-           (namespace is not None and not isinstance(namespace, dict))):
+                (kwds is not None and not isinstance(kwds, dict)) or
+                (namespace is not None and not isinstance(namespace, dict))):
```

Comment:
Please remove _all_ changes not directly relevant to the PR, such as this stylistic change.

Reply:
Ah, great. I have a related question.

So what do I do about stylistic changes?

Can I issue a separate PR for such?

If no, why not?

If yes, can I issue stylistic changes  different parts of library in one PR or should a PR only contain related stylistic changes?

Reply:
> So what do I do about stylistic changes?

Nothing :) Quoting [the devguide, on making good PRs](https://devguide.python.org/getting-started/pull-request-lifecycle/#making-good-prs):

> Pull requests with only code formatting changes are usually rejected. On the other hand, fixes for typos and grammar errors in documents and docstrings are welcome.